.github/CODEOWNERS

```diff
@@ -528,6 +528,12 @@
 # ServiceLabel: %Guest Configuration
 # ServiceOwners:                                                   @mgreenegit @vivlingaiah
 
+# PRLabel: %Hardware Security Modules
```

Comment:
We'll want to move this entire block down to the management section.   Since we're organizing alphabetically by label, it should land around L956.

Reply:
updated